tests/xbps/libxbps/config/main.c

```diff
@@ -39,6 +39,8 @@ ATF_TC_BODY(config_include_test, tc)
 	struct xbps_handle xh;
 	const char *tcsdir;
 	char *buf, *buf2, pwd[PATH_MAX];
+	char confdir[PATH_MAX + 1 + sizeof("xbps.d")];
```

Comment:
why allow confdir to be larger than PATH_MAX? why not cap it?

Reply:
I suspect for the null terminator at least